src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -148,14 +148,16 @@ public int CompareItems(CompletionItem item1, CompletionItem item2, string patte
             var match1 = GetMatch(item1.FilterText, pattern, includeMatchSpans: false, culture);
             var match2 = GetMatch(item2.FilterText, pattern, includeMatchSpans: false, culture);
 
-            return CompareItems(item1, match1, item2, match2);
+            return CompareItems(item1, match1, item2, match2, out _);
         }
 
-        public int CompareItems(CompletionItem item1, PatternMatch? match1, CompletionItem item2, PatternMatch? match2)
+        public int CompareItems(CompletionItem item1, PatternMatch? match1, CompletionItem item2, PatternMatch? match2, out bool onlyDifferInCaseSensitivity)
         {
+            onlyDifferInCaseSensitivity = false;
+
             if (match1 != null && match2 != null)
             {
-                var result = CompareMatches(match1.Value, match2.Value, item1, item2);
+                var result = CompareMatches(match1.Value, match2.Value, item1, item2, out onlyDifferInCaseSensitivity);
```

Comment:
so if result is 0 from this, is it ok that we set this out parameter to some value?  should we set a temp variable isntead, and only assign the out param if result != 0?

Reply:
if result == 0, then `onlyDifferInCaseSensitivity` would be false, so it is still correct.

Reply:
can you add an assert for this.

Reply:
Sure

Reply:
please add assert :)

Reply:
n/m you did!